	Make sure that, as advertised, cef5conv acts as bg5conv + cefconv
	and cefsconv as sjisconv + cefconv.

	Make the TeX description of cefconv agree with the actual code.

diff -ur cjkutils-4.8.2.orig/CEFconv/cef5conv.w cjkutils-4.8.2/CEFconv/cef5conv.w
--- cjkutils-4.8.2.orig/CEFconv/cef5conv.w	2010-12-17 20:41:33.000000000 +0100
+++ cjkutils-4.8.2/CEFconv/cef5conv.w	2011-09-16 21:49:49.000000000 +0200
@@ -172,6 +172,7 @@
                 goto no_macro;@#
 
             *(outp++) = '\177';
+            *(outp++) = '\177';
             *(outp++) = '\"';
             *(outp++) = '0';@#
 
@@ -188,7 +189,6 @@
                 goto no_macro;@#
 
             *(outp++) = '\177';
-            *(outp++) = '\177';
             *(outp++) = '\"';
             *(outp++) = '0';@#
 
diff -ur cjkutils-4.8.2.orig/CEFconv/cefconv.w cjkutils-4.8.2/CEFconv/cefconv.w
--- cjkutils-4.8.2.orig/CEFconv/cefconv.w	2010-12-17 20:41:34.000000000 +0100
+++ cjkutils-4.8.2/CEFconv/cefconv.w	2011-09-17 14:19:53.000000000 +0200
@@ -80,7 +80,7 @@
 IRIZ, a private encoding CY, and U for Unicode encoding; \.{yyzz} is a
 hexadecimal representation of the code point in this plane) with
 \medskip
-\quad\.{\char94 \char94 7f72\char94 \char94 ffXX\char94 \char94 7f%
+\quad\.{\char94 \char94 7f72\char94 \char94 7fXX\char94 \char94 7f%
 \char94 \char94 7f"0yy\char94 \char94 7f"0zz\char94 \char94 7f}\quad.
 \medskip
 \.{XX} is the corresponding CJK encoding of \.{xx}; the number `72'
diff -ur cjkutils-4.8.2.orig/CEFconv/cefsconv.w cjkutils-4.8.2/CEFconv/cefsconv.w
--- cjkutils-4.8.2.orig/CEFconv/cefsconv.w	2010-12-17 20:41:35.000000000 +0100
+++ cjkutils-4.8.2/CEFconv/cefsconv.w	2011-09-16 21:49:46.000000000 +0200
@@ -172,6 +172,7 @@
                 goto no_macro;@#
 
             *(outp++) = '\177';
+            *(outp++) = '\177';
             *(outp++) = '\"';
             *(outp++) = '0';@#
 
@@ -188,7 +189,6 @@
                 goto no_macro;@#
 
             *(outp++) = '\177';
-            *(outp++) = '\177';
             *(outp++) = '\"';
             *(outp++) = '0';@#
 
